aibolit/metrics/countLeaves/numberofleaves.py

```diff
@@ -0,0 +1,45 @@
+import javalang
+from aibolit.patterns.var_middle.var_middle import JavalangImproved
+from typing import List
+
+
+class CountNumberOfLeaves:
+    '''
+    Returns number of leaves for each method in class.
+    input: file_path
+    output: list of numbers. Size of this list == number of methods in class.
+    '''
+    def __init__(self):
+        pass
+
+    def value(self, filename: str):
```

Comment:
Metric has to return a number, not a list of lines. 
```
def value(self, filename: str) -> float:
```

Reply:
> Metric has to return a number, not a list of lines.
> 
> ```
> def value(self, filename: str) -> float:
> ```

fixed